### PR TITLE
Changelog for release v0.98.0

### PR DESCRIPTION
**Pull request by pjanotti**
Changelog updates for release v0.98.0

### CODE REVIEW

File: CHANGELOG.md
Comment (crobert-1) [inline CHANGELOG.md:20]:
```suggestion
- (Contrib) `postgresqlreceiver`: Minimal supported PostgreSQL version will be updated from 9.6 to 12.0 in a future release. ([#30923](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30923))
```

File: CHANGELOG.md
Comment (crobert-1) [inline CHANGELOG.md:25]:
```suggestion
- (Splunk) Add SQL Server receiver ([#4649](https://github.com/signalfx/splunk-otel-collector/pull/4649))
```

File: CHANGELOG.md
Comment (crobert-1) [inline CHANGELOG.md:46]:
```suggestion
- (Contrib) all: Add support for ARM build ([#12920](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/12920))

```
I assume we should add this, or is there a reason we shouldn't advertise this?

**Review by crobert-1 — COMMENTED** [review]
We should add `core` [changes in v0.98.0](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.98.0/CHANGELOG.md#v150v0980) as well.

File: CHANGELOG.md
Comment (pjanotti) [reply CHANGELOG.md:46]: Since we already ship ARM binaries I thought that this one was redundant for us, if that is not the case I will incorporate your suggestion.

**Comment by pjanotti** [comment]
> We should add core [changes in v0.98.0](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.98.0/CHANGELOG.md#v150v0980) as well.

Somehow I did some mistake and when I looked at the release notes for core I got the impression that they were empty... how the heck that happened I don't know... 🤔

**Review by crobert-1 — COMMENTED** [review]
Rebasing should add https://github.com/signalfx/splunk-otel-collector/pull/4658

Should we include https://github.com/signalfx/splunk-otel-collector/pull/4651? https://github.com/signalfx/splunk-otel-collector/pull/4645? https://github.com/signalfx/splunk-otel-collector/pull/4644?

(Honest questions, I didn't review each in-depth enough to know)

**Comment by pjanotti** [comment]
Good questions, I appreciate the extra eyes @crobert-1 

> Rebasing should add https://github.com/signalfx/splunk-otel-collector/pull/4658

Correct

> Should we include https://github.com/signalfx/splunk-otel-collector/pull/4651? https://github.com/signalfx/splunk-otel-collector/pull/4645? https://github.com/signalfx/splunk-otel-collector/pull/4644?

#4651 : will be documented in a batch when other changes benefiting from the MSI new capabilities are documented
#4645 : yeah, this one deservers to be added
#4644 : I assumed that this is more for our support, not sure if we want to add it to the changes read by users - any thoughts @dloucasfx

**Review by crobert-1 — APPROVED** [review]
LGTM, I'll defer to others on the remaining items to know if they're worth adding here or not.

**Review by samiura — APPROVED** [review]

**Comment by samiura** [comment]
@pjanotti this needs to be rebased

File: CHANGELOG.md
Comment (atoulme) [inline CHANGELOG.md:32]:
```suggestion
```

File: CHANGELOG.md
Comment (crobert-1) [inline CHANGELOG.md:63]: I think we should actually move this up to breaking changes, and notify users that a new permission is required to properly get metrics from the Oracle DB receiver.[ Context here](https://github.com/signalfx/splunk-otel-collector/pull/4656#issuecomment-2059380542).

**Comment by pjanotti** [comment]
I'm going to rebase it to ensure that it is correct after all the conflict resolutions, that will be done after OTel update PR is merged.

**Comment by pjanotti** [comment]
gah.. forgot to pull main latest before rebasing...

**Pull request merged**
